app-web/src/components/Cards/Cards.js

```diff
@@ -92,17 +106,27 @@ const Cards = ({ topic, sourcePath, cards }) => {
       </Link>
     );
   if (cardComponents.length > 0) {
+    // find the ideal large screen limit based on number of cards, and cards rendering per row
+    const CARDS_PER_ROW = 3; // ideally in a true grid system we would have a more concrete styling
```

Comment:
I might try constants like this in a `constants.js` file.

Reply:
good point, changed